docs/node-operators/migration-guides/upgrade-to-0710.md

```diff
@@ -0,0 +1,395 @@
+
+# copy wallets
+cp -r <VEGA-NETWORK-HOME>/data/node/wallets <BACKUP-FOLDER>/v0.71.2/wallets
+cp <TENDERMINT-HOME>/node_key.json <BACKUP-FOLDER>/v0.71.2/wallets
+cp <TENDERMINT-HOME>/priv_validator_key.json <BACKUP-FOLDER>/v0.71.2/wallets
+cp <VEGA-NETWORK-HOME>/nodewallet-passphrase.txt <BACKUP-FOLDER>/v0.71.2/wallets  # filename and location might differ, depending on your setup
+
+# copy network state
+cp -r <VEGA-NETWORK-HOME>/state/node <BACKUP-FOLDER>/v0.71.2/core-state
+cp -r <TENDERMINT-HOME>/data <BACKUP-FOLDER>/v0.71.2/tm-state
+
+# copy vegavisor config if you are running Visor on your node
+cp -r <VEGAVISOR-HOME>/current <BACKUP-FOLDER>/v0.71.2/vegavisor-current
+
+# Check if backup has been successfully done*; check if all files has been copied correctly
+tree <BACKUP-FOLDER>
+
+# Backup PostgreSQL if you have been running data node**
+pg_dump --host=localhost --port=5432 --username=<VEGA-DB-USER> --password -Fc -f <BACKUP-FOLDER>/v0.71.2/data_node_db.bak.sql <VEGA-DB-NAME>
+```
+
+**Notes**: 
+* *The `tree` command needs to be installed (e.g. `apt-get install -y tree`) but it is the easiest way to see if backup files match the original files without going too deep into details.
+*  **You might see some errors when running `pg_dump`. To learn if they can be safely ignored, see the [troubleshooting section in the official timescaledb docs ↗](https://docs.timescale.com/timescaledb/latest/how-to-guides/backup-and-restore/troubleshooting/).
+
+
+### 3. Download new binaries
+Download new binaries for the upgrade version from the [GitHub releases page ↗](https://github.com/vegaprotocol/vega/releases) and unzip them. Save them in the `<VEGA-BIN>` and the `<VISOR-BIN>` you chose. 
+
+The binaries you need: 
+* Vega binary: Also includes Tendermint and data node as binary subcommands
+* Visor binary: Optional for setting up Visor for protocol upgrades (See the docs listed in [the previous migration guide](./upgrade-node.md#7-read-the-visor-documentation) for information on Visor.
+
+See example commands for downloading below. You may need to update the version number depending on the version of the binaries you need to update to:
+
+```bash
+# Download archives
+wget https://github.com/vegaprotocol/vega/releases/download/v0.71.2/vega-linux-amd64.zip
+wget https://github.com/vegaprotocol/vega/releases/download/v0.71.2/visor-linux-amd64.zip
+
+# Unzip downloaded archives
+unzip vega-linux-amd64.zip
+unzip visor-linux-amd64.zip
+
+mv vega <VEGA-BIN>
+mv visor <VISOR-BIN>
+```
+
+:::note  Manual update for binaries when running Visor
+Visor cannot automatically restart (or upgrade) binaries because you are restarting the network with a `checkpoint`. Once you stop the Vega network, you must update the old binaries with the downloaded ones. You can find the binary path in the `<VEGAVISOR-HOME>/current/run-config.toml` file. Usually the path is `<VEGAVISOR-HOME>/current/vega`
+:::
```

Comment:
```suggestion
Visor cannot automatically restart (or upgrade) binaries because this release requires restarting the network with a `checkpoint`. Once you stop the Vega network, you must update the old binaries with the downloaded ones. You can find the binary path in the `<VEGAVISOR-HOME>/current/run-config.toml` file. Usually the path is `<VEGAVISOR-HOME>/current/vega`
:::
```